Tests/LeafKitTests/LeafKitTests.swift

```diff
@@ -34,7 +34,7 @@ final class ParserTests: XCTestCase {
 
         let expectation = """
```

Comment:
Various minor internal changes to print formatting of [ParameterDeclaration] for clarity